NeuralAmpModeler/NeuralAmpModeler.cpp

```diff
@@ -734,7 +734,7 @@ void NeuralAmpModeler::_FallbackDSP(iplug::sample **inputs,
 std::string NeuralAmpModeler::_GetNAM(const WDL_String &modelPath) {
   WDL_String previousNAMPath = this->mNAMPath;
   try {
-    auto dspPath = std::filesystem::path(modelPath.Get());
+    auto dspPath = std::filesystem::path(modelPath.Get());    
```

Comment:
Would you please apply [`format.bash`](https://github.com/sdatkinson/NeuralAmpModelerPlugin/blob/main/format.bash) to the PR?